<!DOCTYPE html>
<html lang="en-US">
<!--********************************************-->
<!--*       Generated from PreTeXt source      *-->
<!--*                                          *-->
<!--*         https://pretextbook.org          *-->
<!--*                                          *-->
<!--********************************************-->
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta name="robots" content="noindex, nofollow">
</head>
<body class="ignore-math">
<h4 class="heading"><span class="type">Paragraph</span></h4>
<div class="para">Output from running <a href="" class="xref" data-knowl="./knowl/listing-ex-extest2.html" title="Listing 11.3.4">ListingÂ 11.3.4</a> tells us the class of the exception that was caught and a description of the problem.</div>
<span class="incontext"><a href="section-ex-catching.html#p-848" class="internal">in-context</a></span>
</body>
</html>
